src/Symfony/Component/Form/ChoiceList/View/ChoiceView.php

```diff
@@ -30,10 +30,10 @@ class ChoiceView
     /**
      * Creates a new choice view.
      *
-     * @param mixed  $data  The original choice
-     * @param string $value The view representation of the choice
-     * @param string $label The label displayed to humans
-     * @param array  $attr  Additional attributes for the HTML tag
+     * @param mixed        $data  The original choice
+     * @param string       $value The view representation of the choice
+     * @param string|false $label The label displayed to humans. Pass false to discard the label
```

Comment:
or maybe as a single sentence?
`The label displayed to humans; pass false to discard the label`